crates/metal-codegen-llvm/src/core.rs

```diff
@@ -191,22 +201,21 @@ mod tests {
     fn test_nested_module_function() {
         // NOTE `children` is not appended to cause Rust sucks.
```

Comment:
what an aggressive, grammatically-incorrect statement you've got here! (`> /dev/null` or clarify please)